db/migrations/20190523150829_create_events.rb

```diff
@@ -10,6 +10,7 @@
 
       column :started_at, DateTime
       column :ended_at, DateTime
+      column :state, String, null: false, default: 'unpublished'
```

Comment:
it will be good to make enum type here and use it in tasks table too

Reply:
done